src/Core/Package.vala

```diff
@@ -386,6 +388,37 @@ public class AppCenterCore.Package : Object {
         return latest_version;
     }
 
+    public string? get_author () {
```

Comment:
Can we use properties instead of `get` methods please? It's more vala style to have `package.author`